dns/perf/README.md

```diff
@@ -0,0 +1,151 @@
+# Overview
+
+This directory contains scripts used to run a dns performance test in a
+Kubernetes cluster. The performance script `run` benchmarks the performance of a
+single DNS server instance with a synthetic query workload.
+
+# Quickstart
+
+## Running a performance test
+
+Assuming you have a working `kubectl` command:
+
+``` sh
+$ mkdir out/                                        # output directory
+$ ./run --params params/default.yaml --out-dir out  # run the perf test
+```
+
+`run` will run a performance benchmark ranging over the parameters given in
+`--params`. The included `default.yaml` run will take several hours to run
+through all combinations. Each run will create a `run-<timestamp>` directory
+under the output directory. `latest` symlink will point to the latest run
+directory that was created.
+
+### Benchmarking the cluster DNS
+
+You can benchmark the existing cluster DNS by specifying the `--use-cluster-dns`
+flag. (As opposed to the server referenced by `--deployment-yaml`). Note: you
+should be aware that some noise may be introduced by the client running on the
+same pod as a DNS server.
+
+Note: some parameters are skipped if running with the cluster DNS as they do not
```

Comment:
Should we mention which params will be skipped, or instead say which params will be run?